client/src/components/ListCover.jsx

```diff
@@ -20,15 +20,15 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const ListCover = () => {
+const ListCover = ({ list }) => {
   const classes = useStyles();
-
+  const { title } = list;
   return (
     <Grid item>
       <Card className={classes.card} elevation={0}>
         <CardMedia className={classes.image} image={sample} />
         <CardContent>
-          <Typography align="center">Title</Typography>
+          <Typography align="center">{title}</Typography>
           <Typography variant="subtitle1" align="center">
             50 items
```

Comment:
can we also make this dynamic?

Reply:
Yup sure, I'll make it happen in the next PR😁